src/medicine/medicine_classes.py

```diff
@@ -137,25 +137,37 @@ def append_instance_to_file(self, file_name='instances.p'):
 
         with open(file_name, 'wb') as file:
             pickle.dump(instances_dict, file)
-    
-    
+
+
     def add_medicine(self, amount_of_pills):  # funkcja jeszcze nie gotowa, trzeba ją rozbudować (i to porządnie), tego komentarza nie usuwaj
         """Adds more pills to an instance"""
         assert isinstance(amount_of_pills, int), 'amount_of_pills is supposed to be INT'
         self.current_amount += amount_of_pills
-    
-    
+
+
     def how_much_in_mg(self):
         """Basing on self.current_amount returns amount of the medicine in milligrams"""
         amount_of_pills = self.current_amount
         mg_per_pill = self.dose
         amount_of_mg = amount_of_pills * mg_per_pill
         return amount_of_mg
-    
-    
-    def __str__(self):  # __str__() i __repr__() używam z przyzwyczajenia, tak mi wpojono. W docstringu masz co robią. 
```

Comment:
tak w ogóle to polskie komentarze to moja komunikacja z Mikołajez zwłaszcza jak się domyślam, że czegoś może nie umieć z Pythona

Reply:
to jest też spoko. Ogólnie się używa angielski w komentarzach, a zmienne itp. powinny zawsze być po angielsku, ale nie ma problemu mieć po polsku. Szczególnie że to twój osobisty projekt :D